docs/docs/integrations/language-models/open-ai.md

```diff
@@ -73,14 +73,15 @@ OpenAiChatModel model = OpenAiChatModel.builder()
     .logResponses(...)
     .tokenizer(...)
     .customHeaders(...)
+    .isStreaming(...)
```

Comment:
User should not be forced to set this property. `OpenAiChatModel` should be both sync and streaming at the same time, depending on the invoked method.

Reply:
Now `OpenAiChatModel` implements the `StreamingChatLanguageModel` interface